src-ts/node/crypto.ts

```diff
@@ -0,0 +1,3 @@
+export function createHash() {
+    throw new Error("TODO: Implement crypto.createHash for node");
```

Comment:
why even mark this as TODO?

Reply:
Because we'll need it eventually, but it's not required for the code path use cases I was testing.  They only need the function exist.